include/singa/core/scheduler.h

```diff
@@ -68,6 +76,11 @@ class Node {
   OpFunc op_;
   EdgeVec in_edges_;
   EdgeVec out_edges_;
+
+  string op_name_;
+  float time_elapsed_ = 0;
+  int iteration_ = 0;
```

Comment:
I think we can move this variable to Graph. Although some ops might be executed more times than others when using old APIs.

Reply:
moved the variable to the graph, provide a step() function in graph to increment step counter